admin/system/insert_berita.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+    include("connection.php");
+    session_start();
+
+    //get data
+    $judul_berita = $_POST["judul_berita"];
+    $kategori = $_POST["kategori"];
+    $id = uniqid();
+    mkdir("../../news/".$id);
+
+    if(isset($_FILES["gambar1"])){
+        mkdir("../../news/".$id."/gambar");
+        $gambar = $id;
+        for($i=1; $i <= (int)$_POST["jumlah_gambar"]; $i++){
+            $target = $i.".".strtolower(pathinfo(basename($_FILES["gambar".$i]["name"]),PATHINFO_EXTENSION));
+            $target_dir = "../../news/".$id."/gambar/";
+            $target_file = $target_dir . $target;
+            $imageFileType = strtolower(pathinfo($target_file,PATHINFO_EXTENSION));
+            
+            if (move_uploaded_file($_FILES["gambar".$i]["tmp_name"], $target_file)) {
+                $gambar = $gambar."-".$i.".".$imageFileType;
+            } else {
+                $gambar = null;
+            }
+        }
+    }
+    else{
+        $gambar = null;
+    }
+
+    if(isset($_POST["isi_berita"])){
+        mkdir("../../news/".$id."/isi_berita");
+        $myfile = fopen("../../news/".$id."/isi_berita/".$id.".html", "w") or die("Unable to open file!");
```

Comment:
Codacy found an issue: [Use of die language construct is discouraged.](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5762293)